### PR TITLE
Add (very) short introduction to Miking DPPL and CorePPL to README

### PR DESCRIPTION
**Pull request by dlunde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
-## Getting Started
+## CorePPL
+CorePPL extends MExpr (see [Miking](https://github.com/miking-lang/miking)) with probabilistic constructs for defining random variables and for likelihood updating. For example, the program
+```
+mexpr
+let x = assume (Beta 10.0 5.0) in
+observe true (Bernoulli x);
+x
+```
+encodes a simple distribution for the bias of a coin, by setting a Beta prior for the bias and observing a single flip of the coin (resulting in `true`, i.e. heads).
```
Comment (david-broman) [inline README.md:17]: From earlier discussions, we should probably not use the term `bias` in this context. Suggestion: "... by setting a Beta prior for the probability observing head (resulting in `true`) for a single flip of the coin."

File: README.md [hunk 2]
```diff
@@ -17,14 +47,14 @@ CUDA installation guides: [Linux](https://docs.nvidia.com/cuda/cuda-installation
 To run the CPU version in parallel, [OpenMP](https://www.openmp.org/resources/openmp-compilers-tools/) must be installed. 
 OpenMP comes with recent gcc versions. However, OpenMP is not necessary if one does not want to execute programs in parallel on the CPU.
 
-### Install rootppl
+#### Install rootppl
 To install rootppl for the current user, first clone this repository and change directory to the rootppl folder. Then run:
```
Comment (david-broman) [inline README.md:51]: Change to `rootppl` in typewriter font.

**Comment by david-broman** [comment]
I also found some things that need to be updated in the RootPPL part:

- When we talk about sweeps, it would be good to add a sentence to say what we mean.
- The RootPPL text talks about `pc`. This should be updated to describe how `next` is used.

**Comment by david-broman** [comment]
I just discovered for `midppl`, it is not possible to write
```
make clean
make install
```

you have to write 
```
make clean
make
make install
```
it would be good if we can just write `make install` and it would then build it also.

**Comment by david-broman** [comment]
Besides the above, it worked well to compile and run the example in the latest version.

File: README.md (hunk 1 above)
Comment (dlunde) [reply README.md]: Done, resolving

File: README.md (hunk 2 above)
Comment (dlunde) [reply README.md:51]: Sure, I can change it. Note, however, that I have not made any changes to the RootPPL section (written by Joey).

**Comment by dlunde** [comment]
I have now addressed the comments.

You can now run `make install` and `midppl` will be built, if not built already. The `Makefile` now has a dependency on `find` for establishing dependencies on all `.mc`-files in the repo for when to rebuild `build/midppl`. An alternative is to make `build/midppl` `PHONY`, but then it will always be rebuilt when running `make`, `make install`, etc.

Note that I have not made any modifications to the RootPPL part of the README (written by Joey). Iterating the RootPPL section would also be good, but is not part of this PR. @david-broman Maybe you can make an issue with your comments, so we don't forget?

**Comment by david-broman** [comment]
OK, I can fix it in a later PR. Thanks!

**Pull request merged**